crypto/objects/objects.txt

```diff
@@ -676,6 +676,7 @@ algorithm 17		: DES-EDE		: des-ede
 			: DESX-CBC		: desx-cbc
 algorithm 18		: SHA			: sha
 algorithm 26		: SHA1			: sha1
+			: SHA-1			: sha-1
```

Comment:
Ah, sorry, no.  The proper way to add an alias would be this (granted, the syntax is not the clearest):

```
!Cname sha-1
algorithm 26		: SHA1			: sha1
```

What you have done now is to create a *separate* object called `sha-1`, without the corresponding implementation...